webapp-django/crashstats/signature/jinja2/signature/signature_summary.html

```diff
@@ -276,8 +276,8 @@ <h2>Graphics Adapter</h2>
     <table class="tablesorter data-table">
       <thead>
         <tr>
-          <th class="header">Vendor</th>
-          <th class="header">Adapter</th>
+          <th data-field-name="adapter_vendor" class="header">Vendor</th>
+          <th data-field-name="adapter_device" class="header">Adapter</th>
```

Comment:
These two fields sometimes sorted correctly, but if all the values were hex strings, they wouldn't sort at all. This fixes that by telling the sort-fixing code the names of these fields so we can override the sorter used.